### 6/15 Today I learned...

#### Error Handling
##### Error Handling features give programmers how much information we wanna share with users when things go wrong.
An example.
```py
a = input("Enter your number: ")
b = input("Enter your second number: ")

try:
    a = int(a)
    b = int(b)
    c = a/b
except ValueError:
    print("You should type a number, try again!")
except ZeroDivisionError:
    print("You can't divide any number by 0")
else:
    print(c)
```

We put the codelines to test under `try:` block.

Beside `except`, we specify errors that might appear from the `try:` block.

Also, under the `except` block, we put what we will do if the specified error occurs.

In this example, if `ValueError` occurs, we print `You should type a number, try again!`

Or we can just silently pass the failing, by saying 

```py
except NameError:
    pass
```

Then when NameError arises, it will be silent on the surface.
